### DISCUSSION TITLE
177

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#142] [General]
# 潮流周刊第177期 - 五彩花海

封面图拍摄来源于上次在径山花海随手拍的各种颜色的花，这是杭州的一个好地方，地方大，人不多，旁边好停车的地方，经常会有应季的花看看，不过我更喜欢凉快的天气弄一个垫子躺着吹风。

https://weekly.tw93.fun/posts/177-%E4%BA%94%E5%BD%A9%E8%8A%B1%E6%B5%B7

**Comment by eynol** [comment]
博主每月在CDN的费用大概是多少呢，我看网站的图片加载很快

**Reply by tw93** [reply]
更多是本身做了很多优化，包括裁剪懒加载等工作

**Comment by tonytian2000** [comment]
楼主能帮忙宣传下我的网站么？我还收录了你的RSS，谢谢了
www.zero2me.com
https://www.zero2me.com/hotNews/0

**Comment by Spike-Leung** [comment]
所以那个瓜甜吗 :P

**Reply by tw93** [reply]
当时没买西瓜，买榴莲了
